test/openapi3.test.js

```diff
@@ -75,6 +78,38 @@ describe('openapi3 tests', () => {
             openapi3.fakeBodyParameter(goodData);
             assert(goodData.parameters.length === 1);
         });
+    });
+
+    describe('getTagGroup', () => {
+        let test = null;
+        it('should handle no tag groups', () => {
+            assert.doesNotThrow(() => openapi3.getTagGroup('fake tag', null));
+        });
+        it('should handle wrong tags ', () => {
+            assert.doesNotThrow(() => openapi3.getTagGroup('fake tag', exampleEnv.tagGroups));
+            let result = openapi3.getTagGroup('fake tag', exampleEnv.tagGroups);
+            assert(result.name === 'fake tag');
```

Comment:
So we get better error messages, I think these `assert` calls should use one of the `*Equals` methods e.g. [assert.deepStrictEqual](https://nodejs.org/api/assert.html#assert_assert_deepstrictequal_actual_expected_message)

Reply:
No worries, just seeing this now. Will fix this asap